seed/static/seed/js/controllers/menu_controller.js

```diff
@@ -61,13 +61,23 @@ angular.module('BE.seed.controller.menu', [])
       });
       $scope.$on('app_success', function () {
         $rootScope.route_load_error = false;
-      })
+      });
       $scope.$on('organization_list_updated', function () {
         init();
       });
 
       $scope.is_active = function (menu_item) {
-        if (menu_item === $location.path()) {
+        if ($state.current.url !== '^') {
```

Comment:
$digest iteration error occurred for location search if current url is undefined on start.